operator/pkg/nodeipam/nodesvclb.go

```diff
@@ -185,20 +199,49 @@ func (r *nodeSvcLBReconciler) getRelevantNodes(ctx context.Context, svc *corev1.
 		}
 	}
 
+	return selectedNodes, nil
+}
+
+// getRelevantNodes get all the nodes in the matching EndpointSlices for a specified service
```

Comment:
This one is not introduced by your PR, but let's fix anyway.

```suggestion
// getRelevantNodes gets all the nodes in the matching EndpointSlices for a specified service.
```